projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+| --clr-datagrid-row-hover-font-color                   | Changes the row text color of data table when hovers over element.           |
+| --clr-datagrid-action-toggle-color                    | Changes the toggle action button color.                                      |
+| --clr-datagrid-pagination-btn-color                   | Changes the table bottom pagination buttons color.                           |
+| --clr-datagrid-pagination-btn-disabled-color          | Changes the table bottom pagination buttons color when disabled.             |
+| --clr-datagrid-pagination-input-border-color          | Changes the text box border color of table datagrid.                         |
+| --clr-datagrid-pagination-input-border-focus-color    | Changes the text box border color of table datagrid when focus on element.   |
+| --clr-datagrid-popover-bg-color                       | Changes the background color of popup box.                                   |
+| --clr-datagrid-popover-border-color                   | Changes the border color of popup box.                                       |
+| --clr-datagrid-row-selected                           | Changes the text color of table row when user selected.                      |
+| --clr-datagrid-loading-background                     | Changes the background color of datagrid when click on toggle spinner.       |
+| --clr-datagrid-popovers-box-shadow-color              | Changes the popup box shadow color of datagrid.                              |
+| --clr-datagrid-column-switch-header-font-color        | Changes the column header text color of datagrid.                            |
+| --clr-datagrid-column-switch-header-font-hover-color  | Changes the column header text color of datagrid when hovers over element.   |
+| --clr-datagrid-detail-caret-icon-open-bg-color        | Changes the caret icon background color of datagrid.                         |
+| --clr-datagrid-detail-caret-icon-open-icon-color      | Changes the caret icon color of datagrid.                                    |
+| --clr-datagrid-placeholder-color                      | Changes the text color of datagrid placeholder.                              |
+| --clr-datagrid-placeholder-font-size                  | Changes the placeholder text size of datagrid.                               |
+| --clr-datagrid-pagination-btn-disabled-opacity        | Changes the opacity of pagination button when disabled.                      |
+| --clr-datagrid-column-toggle-border-color             | Changes the toggle button border color.                                      |
+| --clr-datagrid-column-toggle-fill-color               | Changes the background color of toggle button.                               |
+| --clr-datagrid-column-toggle-text-color               | Changes the text color of pagination toggle button.                          |
```

Comment:
There is nothing about pagination here. We're still on the "column toggle" feature.

Reply:
As per your suggestions, changed it.